pkg/scaffold/v1/controller/add.go

```diff
@@ -38,9 +37,9 @@ type AddController struct {
 // GetInput implements input.File
 func (a *AddController) GetInput() (input.Input, error) {
 	if a.Path == "" {
-		a.Path = filepath.Join("pkg", "controller", fmt.Sprintf(
-			"add_%s.go", strings.ToLower(a.Resource.Kind)))
+		a.Path = filepath.Join("pkg", "controller", "add_%[kind]s.go")
```

Comment:
IHMO: -1 for this change. In POV it makes the manutence ability be lower and do not improve the performance and/or design either.